keylime/cloud_verifier_common.py

```diff
@@ -36,8 +36,10 @@ def process_quote_response(agent, ima_policy, json_response, agentAttestState) -
     failure = Failure(Component.QUOTE_VALIDATION)
     received_public_key = None
     quote = None
+    agent_id = "NA"
```

Comment:
Why introduce a magic constant here instead of using None?

Reply:
Could be None, sure